subprojects/core/src/integTest/groovy/org/gradle/initialization/CommandLineArgDeprecationIntegrationTest.groovy

```diff
@@ -32,8 +33,29 @@ class CommandLineArgDeprecationIntegrationTest extends AbstractIntegrationSpec {
 
         where:
         issue                                          | deprecatedArgs        | message
-        'https://github.com/gradle/gradle/issues/1425' | '--recompile-scripts' | StartParameterBuildOptions.RecompileScriptsOption.DEPRECATION_MESSAGE
-        'https://github.com/gradle/gradle/issues/3077' | '--no-rebuild'        | StartParameterBuildOptions.NoProjectDependenciesRebuildOption.DEPRECATION_MESSAGE
-        'https://github.com/gradle/gradle/issues/3077' | '-a'                  | StartParameterBuildOptions.NoProjectDependenciesRebuildOption.DEPRECATION_MESSAGE
+        'https://github.com/gradle/gradle/issues/1425' | '--recompile-scripts' | '--recompile-scripts has been deprecated and is scheduled to be removed in Gradle'
+        'https://github.com/gradle/gradle/issues/3077' | '--no-rebuild'        | '--no-rebuild/-a has been deprecated and is scheduled to be removed in Gradle'
```

Comment:
:x: I'd make the messages constants in the test class. They are repeated multiple times.